paper-menu-button.html

```diff
@@ -37,7 +37,7 @@
 
     <paper-menu-button>
       <paper-icon-button icon="menu" class="dropdown-trigger"></paper-icon-button>
```

Comment:
`class="dropdown-trigger"` -> `slot="dropdown-trigger"`
Also the comment above still mentions using classes instead of slots.